src/js/index.js

```diff
@@ -8,3 +8,4 @@ import './modal';
 import './oops';
 import './buttons-icons';
 import './api-home';
+import './api-catalog-page';
```

Comment:
import './api-catalog-page';  
nie może być w index.js bo to jest do catalogu tylko, wywalam to przez edit